src/reducers/index.js

```diff
@@ -0,0 +1,38 @@
+import { bindActionCreators } from "redux";
+
+// Export Two Things - initialState, reducer fn
+
+const initialState = {
+    additionalPrice: 0,
+    car: {
+      price: 26395,
+      name: '2019 Ford Mustang',
+      image:
+        'https://cdn.motor1.com/images/mgl/0AN2V/s1/2019-ford-mustang-bullitt.jpg',
+      features: []
+    },
+    store: [
+      { id: 1, name: 'V-6 engine', price: 1500 },
+      { id: 2, name: 'Racing detail package', price: 1500 },
+      { id: 3, name: 'Premium sound system', price: 500 },
+      { id: 4, name: 'Rear spoiler', price: 250 }
+    ]
+  };
+  
+  export const carReducer = (state = initialState, action) => {
+    switch (action.type) {
+        case 'REMOVE_FEATURE':
```

Comment:
Okay, so when you're removing a feature in this project, three things are supposed to happen: you're setting the additional price to the additional price minus the price of the feature you're removing, you're filtering out the feature from the car's features, and you're adding it back to the store.